app/src/main/java/currencyConverter/converter/CurrencyRateConverter.java

```diff
@@ -0,0 +1,38 @@
+package currencyConverter.converter;
+
+import java.text.ParseException;
```

Comment:
Is there any special intent of placing java stanard packages before currencyConverter packages? If no, I think it's better to swap them, because currencyConverter is more important in current context, then lists and arrays.

Reply:
The import statements must follow the package statement. import statements should be sorted with the most fundamental packages first, and grouped with associated packages together and one blank line between groups.
Look forward: https://petroware.no/javastyle.html

Reply:
Ok. Absolutely opposite for C++ which I'm familiar with.